home-assignments/session1/ex1_ages_groups.py

```diff
@@ -0,0 +1,46 @@
+import json
+import yaml
+
+
+with open(r'C:\Users\HEZI LEVI\PycharmProjects\ops_assingment1\ppl_ages') as f:
```

Comment:
1. Why file opening happens here? It should be under the main section
2. The path of the file should be relative to the directory where the program is running - I wasn't able to run this program because I have Mac - no "c:/" (The best thing will be to get these as parameters: file_path and file_name, but this is more intermediate) 

```suggestion
with open(str(os.path.dirname(os.path.realpath(__file__)) + "/ex1_ppl_ages" )) as f:```